src/main/java/com/foundation/salesforce/entities/Task.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * @(#) Task.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.entities;
+
+/**
+ * Task
+ *
+ * @author Alejandro Sánchez Luizaga
```

Comment:
Please put point at the end of your name.

Reply:
Do we have to place a dot at the end of @author and @version annotations?
Personally, I guess not. Please, let me know if otherwise.